packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -910,7 +919,8 @@ export abstract class FluidDataStoreContext
 		existing: boolean,
 	): Promise<void> {
 		if (this.channel) {
-			throw new Error("Runtime already bound");
+			return;
```

Comment:
Can you please add comment why not throwing is correct here?
I'd remove commented line